src/presence_analyzer/templates/presence_start_end.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+{% block head %}
+    {% block title %}<title>Presence analyzer</title>{% endblock %}
```

Comment:
`<title>` shouldn't be wrapped in this block. No need for repeating this in all templates.